pkg/metastore/ms_client.go

```diff
@@ -201,7 +224,7 @@ func Merge(ctx context.Context, table *Table, partitionIter Collection, toDB, to
 	err = DiffIterable(partitionIter, toPartitionIter, func(difference catalog.DifferenceType, value interface{}, _ string) error {
 		partition, ok := value.(*Partition)
 		if !ok {
-			return fmt.Errorf("%w in diffIterable call. expected to get * Partition, but got: %T", ErrExpectedType, value)
+			return fmt.Errorf("%w in diffIterable call. expected to get * Partition, but got: %T", msErrors.ErrExpectedType, value)
```

Comment:
maybe something like - suggestion:
```suggestion
			return fmt.Errorf("%w at DiffIterator, got '%T' while expected '*Partition'", msErrors.ErrExpectedType, value)
```